torch/csrc/jit/tensorexpr/llvm_codegen.cpp

```diff
@@ -550,10 +550,13 @@ void LLVMCodeGenImpl::emitKernel(
 
   optimize(*module_);
 
-  // print graph debug info after optimization
   asmBuffer.set_size(0);
   module_->print(asmStream, nullptr);
   llvmCode = asmStream.str().str();
+  GRAPH_DEBUG(
+      "\nLLVM module after optimizations\n\n", asmStream.str().str(), "\n");
+
+  // print graph debug info after optimization
```

Comment:
nit: the comment is moved around by accident? It is supposed to appear in front of the all prints. Another small thing, probably use `llvmCode` instead of `asmStream.str().str()` inside `GRAPH_DEBUG`?